specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/CommonDefinitions.json

```diff
@@ -3262,7 +3262,11 @@
```

Comment:
>GET/PUT [](http://example.com/codeflow?start=119&length=7)

nit for all changes:

GET/PUT is a bit more technical. Can you say "create or read" requests? is it also not present in collection GET type of requests etc.

Reply:
Fixed